### ISSUE TITLE
calculate_risk_percentiles function raises ZeroDivisionError

### ISSUE THREAD

**Issue by yamasakih** [yamasakih/predictiveness-curve#58]
```python
    def test_calculate_risk_percentiles_for_TPR():
        risks = np.array([1.0, 0.89, 0.44, 0, 0.78, 0.67, 0.56, 0.33, 0.22, 0.11])
        points = np.array([0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
        calculate_risk_percentiles = np.frompyfunc(
            lambda p: np.count_nonzero(risks <= p) / len(risks), 1, 1)
        actual = calculate_risk_percentiles(points)
        expect = np.array([0.0, 0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 1.0])
>       np.testing.assert_almost_equal(actual, expect, decimal=4)
E       ZeroDivisionError: float division by zero
```

**Comment by yamasakih** [comment]
`np.count_nonzero(risks <= p)` can be zero.

So when `np.count_nonzero(risks <= p)` is zero, `calculate_risk_percentiles` function should return zero.

**Comment by yamasakih** [comment]
Closes #59

**Comment by yamasakih** [comment]
Fix `calculate_risk_percentiles` function by adding if statement to it to handle `np.count_nonzero(risk >= p)` = 0. https://github.com/yamasakih/predictiveness-curve/pull/59/commits/b41605db00ecab8e28028b606decf8f3ccebe457

**Comment by yamasakih** [comment]
It worked well. I merged #59 .
